### PR TITLE
Update environment handler to apply the update for parameters

### PR DESCRIPTION
**Pull request by afjsilva**
Update environment handler to apply the update for parameters. With this, the host will merge the parameters instead of overwriting.

### CODE REVIEW

**Comment by vitorpmoreira** [comment]
**On-Hold**: Feature not working properly since the parameters are still being overwritten. This is a limitation in the devops center that needs to be addressed a-priori.

**Comment by vitorpmoreira** [comment]
**On-Hold**: Feature not working properly since the parameters are still being overwritten. This is a limitation in the devops center that needs to be addressed a-priori.

**Review by vitorpmoreira — COMMENTED** [review]
Changes approved! Waiting on the Customer Portal PR and Release.

File: src/Version.props [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <Project>
   <PropertyGroup>
-    <Version>1.13.12</Version>
+    <Version>1.13.13</Version>
```
Comment (afjsilva) [inline src/Version.props:3]: I think we don't need to do this version bump. We already have one version bump for v1.13.12 (the latest release was 1.13.11)

**Review by vitorpmoreira — APPROVED** [review]

**Review by vitorpmoreira — APPROVED** [review]

**Pull request merged**